.github/workflows/build-and-deploy-prod.yml

```diff
@@ -41,14 +46,23 @@ jobs:
                   # ref: 'master'
                   path: 'deploy/'
 
-            - name: Build, tag, and push image to Amazon ECR
+            - name: Build image
+              uses: depot/build-push-action@v1
+              with:
+                  context: .
+                  file: prod.web.Dockerfile
+                  load: true
```

Comment:
At the moment, this change reflects the previous behavior, ordering the steps like so:

1. Build the image, loading the result in local Docker
2. Push the image to ECR
3. Extract static assets by copying out of the built container

With Depot, I think it would be faster to reorder it such that the image it pushed to ECR during build (via the remote builder), then pulled locally for asset extraction:

1. Build the image and push to ECR from the remote builder
2. `docker pull` the image from ECR
3. Extract static assets by copying out of the built container

That would add some additional network transfer costs, since you'd be pulling the container from ECR, if that's a concern, hence why I did not make that change. There might also be a way to extract and push the static assets as part of the Docker build itself, to avoid needing to pull the container again at all.